Src/Tools/DiscoveryDocPatcher/Program.cs

```diff
@@ -41,7 +41,7 @@ static int Main(string[] args)
 
         static void PatchDirectory(string rootPath)
         {
-            var patcher = IfFileExists(() => Patcher.Load(Path.Combine(rootPath, "admin_directory_v1.json")));
+            var patcher = IfFileExists(() => Patcher.Load(Path.Combine(rootPath, "admin.directory_v1.json")));
```

Comment:
Should this be `admin.directory.v1` instead?

Reply:
Nope, it matches the structure in https://github.com/googleapis/discovery-artifact-manager/tree/master/discoveries

Reply:
(Entirely reasonable expectation, but...)